parcel/Example_Frostenberg_Immersion_Freezing.jl

```diff
@@ -0,0 +1,175 @@
+colors = [:blue, :green, :darkorange]
+
+function plot_results(
+    sol,
+    variable,
+    label = "",
+    unit = "",
+    linestyle = :solid,
+    color = :black,
+)
+
+    MK.lines!(
+        ax1,
+        sol.t / 60,
+        S_i.(tps, sol[3, :], sol[1, :]) .- 1,
+        label = label * string(variable) * unit,
+        linestyle = linestyle,
+        color = color,
+    )
+    MK.lines!(
+        ax2,
+        sol.t / 60,
+        sol[3, :],
+        label = label * string(variable) * unit,
+        linestyle = linestyle,
+        color = color,
+    )
+    MK.lines!(
+        ax3,
+        sol.t / 60,
+        sol[6, :] * 1e3,
+        label = label * string(variable) * unit,
+        linestyle = linestyle,
+        color = color,
+    )
+    MK.lines!(
+        ax4,
+        sol.t / 60,
+        sol[5, :] * 1e3,
+        label = label * string(variable) * unit,
+        linestyle = linestyle,
+        color = color,
+    )
+    MK.lines!(
+        ax5,
+        sol.t / 60,
+        sol[8, :],
+        label = label * string(variable) * unit,
+        linestyle = linestyle,
+        color = color,
+    )
+    MK.lines!(
+        ax6,
+        sol.t / 60,
+        sol[9, :],
+        label = label * string(variable) * unit,
+        linestyle = linestyle,
+        color = color,
+    )
+
+    MK.axislegend(ax2, position = :lt)
+end
+
+#Frostenberg_mean
+params = parcel_params{FT}(
+    const_dt = const_dt,
+    w = w,
+    aerosol = aerosol,
+    heterogeneous = "Frostenberg_mean",
+    condensation_growth = condensation_growth,
+    deposition_growth = deposition_growth,
+    size_distribution = DSD,
+)
+# solve ODE
+sol = run_parcel(IC, FT(0), t_max, params)
+plot_results(sol, "mean")
+
+
+# Frostenberg_random with different drawing frequencies
+drawing_interval_range = range(FT(1), stop = FT(5), length = 3)
+
+for (drawing_interval, color) in zip(drawing_interval_range, colors)
+
+    params = parcel_params{FT}(
+        const_dt = const_dt,
+        w = w,
+        aerosol = aerosol,
+        heterogeneous = "Frostenberg_random",
+        condensation_growth = condensation_growth,
+        deposition_growth = deposition_growth,
+        size_distribution = DSD,
+        drawing_interval = drawing_interval,
+    )
+    # solve ODE
+    sol = run_parcel(IC, FT(0), t_max, params)
+    plot_results(sol, drawing_interval, "t_d=", " s", :dot, color)
+
+end
+
+
+# Frostenberg_stochastic with different timescales γ
+γ_range = range(FT(1), stop = FT(5), length = 3)
+
+for (γ, color) in zip(γ_range, colors)
+
+    params = parcel_params{FT}(
+        const_dt = const_dt,
+        w = w,
+        aerosol = aerosol,
+        heterogeneous = "Frostenberg_stochastic",
+        condensation_growth = condensation_growth,
+        deposition_growth = deposition_growth,
+        size_distribution = DSD,
+        γ = γ,
+    )
+    # solve ODE
+    sol = run_parcel(IC, FT(0), t_max, params)
+    plot_results(sol, γ, "γ=", " s", :solid, color)
+
+end
+
```

Comment:
We don't need to include the actual `svg` file in the PR. It should be automatically generated when the documentation is build. Once you add a short description of the example to the `docs/src/IceNucleationParcel0D.md` file you should do something like what we do for the other plots there at the end of the file. (Sorry, I tried to paste the code snippet, but github tries to render it and fails)